ocaml/xapi/extauth_plugin_ADpbis.ml

```diff
@@ -20,6 +20,69 @@ open D
 
 open Stdext.Xstringext
 
+let domain_join_cli_path = "/opt/pbis/bin/domainjoin-cli"
```

Comment:
you might want to add this to `Xapi_globs` and include it one of the lists of executables (see `Xapi_globs.ml:Resources`)

Reply:
> you might want to add this to `Xapi_globs` and include it one of the lists of executables (see `Xapi_globs.ml:Resources`)

updated, please have a look. thanks